main.go

```diff
@@ -329,6 +329,9 @@ func runWithOptions(opts Options, out output, logger *zap.Logger) {
 	if err != nil {
 		out.Fatalf("Failed while parsing input: %v\n", err)
 	}
+	if serializerWithClose, ok := serializer.(io.Closer); ok {
+		defer serializerWithClose.Close()
+	}
```

Comment:
It looks a little odd to create a serializer, and close it without doing anything with it. Should we rename the method so it's not `Close()`?

Reply:
Serializer is (deferred) closed after the benchmark, or ad-hoc request handling is complete, `io.Closer` looked cleaner to me as serializer held protobuf descriptor fd (net/file) resource which must be closed after usage. Do you think it must be named something else like a `stop`, `stopDescriptor`, `closeDescriptor`, etc.?

Reply:
Misread the `defer`, so it looked odd. Given the defer, this makes sense to me.